api/build/skip.go

```diff
@@ -13,25 +13,25 @@ import (
 func SkipEmptyBuild(p *pipeline.Build) string {
 	if len(p.Stages) == 1 {
 		if p.Stages[0].Name == "init" {
-			return "skipping build since only init stage found"
+			return "skipping build since only init stage found. It is likely no rulesets matched for the webhook payload."
```

Comment:
Super minor thing: I think we follow Go convention here (even though it never ends up in a proper `error`) for lower casing error message and no periods. So it makes it a little awkward when you add a proper sentence and punctuation. Should we upper case the beginning or lower case everything and pick something like `-` to separate?